index.js

```diff
@@ -59,19 +59,21 @@ app.use('/healthcheck', require('express-healthcheck')({
 }));
 
 const asyncWrap = fn => (req, res, next) => Promise.resolve(fn(req, res, next)).catch(err => next(err));
+const encodedPassword = bcrypt.hashSync(config.PRESHARED_KEY, 10);
 
 app.post('/', asyncWrap(async (req, res) => {
-  const context = {
-    log: logger
-  };
-  //try {
+  const authorization = (req.get('Authorization') || '').replace(/^Bearer /, '');
```

Comment:
Shouldn’t this be Token not bearer?

Reply:
https://www.iana.org/assignments/http-authschemes/http-authschemes.xhtml

I can't find any official docs that mention token

Bearer essentially means "give access to anyone who bears this token"

I think bearer is right but it also apparently doesn't matter